sdk/storage/Azure.Storage.Files.Shares/src/Sas/ShareSasBuilder.cs

```diff
@@ -81,6 +81,19 @@ public class ShareSasBuilder
         /// </summary>
         public string FilePath { get; set; }
 
+        /// <summary>
+        /// Specifies which resources are accessible via the shared access
+        /// signature.
+        ///
+        /// Specify f if the shared resource is a file. This grants access
+        /// to the content and metadata of the file.
+        ///
+        /// Specify s if the shared resource is a share. This grants access
```

Comment:
Typos with the `f` and `s`.

Reply:
@tg-msft, what is the typo?  This was taken directly from the [documentation](https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas), and matched the style of the comment for [BlobSasBuilder.Resource](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs#L99).

Reply:
Oh, sorry, I thought those were weird copy/paste issues.  I forgot these are single character values.  Maybe we should update all the doc examples to include quotes like `"f"` and `"s"` for users because [our generated docs are a little hard to read](https://docs.microsoft.com/en-us/dotnet/api/azure.storage.sas.blobsasbuilder.resource?view=azure-dotnet#Azure_Storage_Sas_BlobSasBuilder_Resource).

Reply:
Fixed in BlobSasBuilder, ShareSasBuilder, and DataLakeSasBuilder.  QueueSasBuilder doesn't have the Resource property.